app/controllers/api/base_controller.rb

```diff
@@ -36,6 +36,11 @@ class BaseController < ApplicationController
                          :only => [:show, :update, :destroy, :handle_options_request, :options]
     end
     skip_before_action :get_global_session_data
+    skip_before_action :reset_toolbar
+    skip_before_action :set_session_tenant
+    skip_before_action :set_user_time_zone
+    skip_before_action :set_gettext_locale
```

Comment:
@abellotti also, we claim not to support `Accept-Language` here - does that need to be updated?